ci_scripts/create_release.rb

```diff
@@ -56,7 +56,16 @@ def run_download_localized_strings
 def create_pr
   # Create a new pull request from the branch
   pr_body = %{
+  - [ ] Verify CHANGELOG
+    - [ ] Entries are intended to be read by humans. Does each entry make sense?
```

Comment:
These are already reviewed, why review them again?

Reply:
Ah, true, that should be done during code review, lets remove that and replace with make sure change log is not empty.